parser_library/include/protocol.h

```diff
@@ -361,7 +361,7 @@ struct PARSER_LIBRARY_EXPORT variable
     explicit variable(const debugging::variable& impl);
 
     sequence<char> name;
-    sequence<char> value;
+    std::string value;
```

Comment:
This should remain non-owning `sequence<char>` "client" view into something that's owned inside the library.